packages/sdk-react/package-lock.json

```diff
@@ -13,6 +13,350 @@
         "@jridgewell/trace-mapping": "^0.3.0"
       }
     },
+    "@azure/abort-controller": {
```

Comment:
what's this ?

Reply:
It should be the dependency of @microsoft/teamsfx@0.7.0. It's generated after running `npm run setup`.

Reply:
This PR only change the teamsfx verison to a specific version, so the deps version not changed, why the lock file updates these deps.

Reply:
It helps to fix the UT failure because `@microsoft/teamsfx@0.7.0` was missing in package-lock.json

Reply:
Reverted the changes after `lerna clean`.